roles/neutron-common/defaults/main.yml

```diff
@@ -11,16 +11,25 @@ neutron:
   service_plugins:
     - neutron.services.l3_router.l3_router_plugin.L3RouterPlugin
   enable_flat_networks: False
-  alembic_version: 0.7.0
 
   # Tunneling encapsulations enabled e.g. gre, vxlan
   # Including any tunnels will enable tunnling
   tunnel_types: []
 
+  lbaas:
+    enable: False
+    interface_driver: neutron.agent.linux.interface.BridgeInterfaceDriver
+    service_plugin:
+      - neutron_lbaas.services.loadbalancer.plugin.LoadBalancerPluginv2
+    service_provider: LOADBALANCERV2:Haproxy:neutron_lbaas.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default
+
   service:
     envs: []
 
   source:
+    python_requirements:
+      - { name: alembic, version: 0.7.0 }
```

Comment:
this is left over  from it trying to use an old version of alembic ...  you might want to remove  this and test if its still required. might  be a nice cleanup item.